popup.css

```diff
@@ -0,0 +1,498 @@
+.expandcard {
+  display: none;
+  background-color: rgb(193, 199, 208, 0.5);
+  font-family: "Poppins", sans-serif;
+  backdrop-filter: blur(5px);
+  position: fixed;
+  padding-top: 15px;
+  top: 0;
+  right: 0;
+  width: 100vw;
+  height: 100vh;
+}
+
+.container {
+  width: 100%;
+  margin: 114px auto;
+  display: grid;
+  gap: 88px;
+  font-family: "Poppins", sans-serif;
+}
+
+.card-body {
+  border-radius: 16px;
+  background-color: #fff;
+  padding: 16px;
+  width: 86%;
+  margin-top: 10%;
+  margin-bottom: 10%;
+  font-family: "Poppins", sans-serif;
+}
+
+.card-body:hover {
+  animation-name: card-body;
+  animation-duration: 3s;
+}
+
+@keyframes card-body {
+  from {
+    background-color: #e5e9f2;
+  }
+
+  to {
+    background-color: #fff;
+  }
+}
+
+/* Popup Styling start */
+.modal-body {
+  position: relative;
+  width: 320px;
+  height: 90%;
+  margin: 0 auto;
+  padding: 16px;
+  background-color: #fff;
+  border-radius: 16px;
+  overflow-x: hidden;
+  z-index: 5;
+  margin-top: -2.8rem;
+}
```

Comment:
- [ ] To make the `pop-up` menu responsive across all `screen` sizes kindly consider using the percentage`(%)` unit for your `width` instead of pixel`(px)`. This approach ensures that the menu adapts and scales appropriately based on the available screen width, offering a better user experience across different devices and resolutions. By using percentage-based widths, the pop-up menu will be more responsive and fluid, adjusting its size proportionally to fit various screen sizes.
`HINT`: Set `width: 80%`;